### ISSUE TITLE
Request: Contensis

### ISSUE THREAD

**Issue by nflatley-zengenti** [simple-icons/simple-icons#11910]
### Brand Name

Contensis

### Website

https://www.contensis.com/

### Popularity Metric

Global rank
#268,088
Country rank
[United Kingdom](https://pro.similarweb.com/#/industry/topsites/All/826/1m)
#19,038
Industry rank
[.../Universities_and_Colleges](https://pro.similarweb.com/#/industry/topsites/Science_and_Education~Universities_and_Colleges/826/1m)
#3,497

### Official Resources for Icon and Color

- SVG: https://github.com/contensis/cli/raw/refs/heads/main/assets/contensis-logo--minimal.svg
- Color: #37BFA7

**Comment by Ahmed20150** [comment]
Hi all, Im still a beginner in Open-Source Contributions and I would love to contribute to your project and to this Issue in specific, can someone let me know if this Issue is still open and if so, what can I do to get started? Thanks!

**Comment by nflatley-zengenti** [comment]
Yes the issue is still open, the SVG link and color above are still valid. Hopefully someone with the knowledge of this org/repo can help you with contribution approach and guidelines. Many thanks.

**Comment by ahmedmahfoudhi** [comment]
I added a [PR](https://github.com/simple-icons/simple-icons/pull/14368) but I believe we need an explicit permission from your organization.

**Comment by sihoran** [comment]
Hi @ahmedmahfoudhi, thanks for raising the PR.

I can confirm I’m authorised to grant permission on behalf of Zengenti Limited (the company behind the Contensis CMS). Please consider this an explicit approval for you to include and distribute the Contensis icon (and any accompanying brand assets included in the PR) in this repository as part of your project, subject to the repository’s licence and contribution process.

Zengenti owns/controls the rights to the Contensis brand assets being submitted, and we’re happy for you to use the submitted icon for this purpose.

If you need this permission in a specific format (or a particular wording for your contributing/CLA requirements), let me know and I’ll provide it.

Simon Horan
Director, Zengenti Limited
